drivers/adodb-mysqli.inc.php

```diff
@@ -946,7 +1158,7 @@ function SetCharSet($charset_name)
 }
 
 /*--------------------------------------------------------------------------------------
-	 Class Name: Recordset
+	 Class Name: ADORecordSet_mysqli
 --------------------------------------------------------------------------------------*/
```

Comment:
This should be a PHPDoc block too; and consider adding one for class _ADODB_mysqli_ too.
```
/**
 * Class ADORecordSet_mysqli
 */
```